test/unit/query-generator/add-column-query.test.ts

```diff
@@ -0,0 +1,35 @@
+import { DataTypes } from '@sequelize/core';
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+describe('QueryGenerator#addColumnQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+  }, { timestamps: false });
+
+  it('generates a ADD COLUMN query in supported dialects', async () => {
```

Comment:
I might be missing something, but why is this `async`?

Reply:
Actually you're right lol. I write a lot of async tests at work so it has become a habit!